extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpSecurityRecorder.java

```diff
@@ -110,6 +110,7 @@ public void formAuthPostHandler(RuntimeValue<Router> httpRouter) {
         if (config.formAuthEnabled()) {
             httpRouter.getValue()
                     .post(config.formPostLocation())
+                    .order(-1 * SecurityHandlerPriorities.FORM_AUTHENTICATION)
```

Comment:
This is actually the only change of this PR, the rest is a refactoring to expose the constants in the runtime module.